### ISSUE TITLE
Wrong talkback speaking in korean label

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#216]
```
What steps will reproduce the problem?
1.In english label "showing items 1 to %d of %d check box not checked"
  -> it's ok
2. In korean label "1개의 항목 중 %d-%d 표시 중 확인란 선택안함"
  -> it's NG
3. In korean label "%d개 항목 체크박스 선택 안 함"
  -> change required it.

What is the expected output? What do you see instead?
==> expected output is above #3 case."%d개 항목 체크박스 선택 안 함"

What version of the product are you using? On what operating system?
==> JB 4.1.x GMS R4 (Talkback ver 3.2.1)

Please provide any additional information below.
This issue is cause by talback because talkback string resource is wrong define.
<string name="template_scroll_from_to_count" 
msgid="3686789962835061906">"<xliff:g id="TOTAL_COUNT">%d</xliff:g>개 항목 
중 <xliff:g id="FROM_INDEX">%d</xliff:g> - <xliff:g id="TO_INDEX">%d</xliff:g> 
표시 중"</string>

Google use "fromIndex -> toIndex -> itemCount" in english but use "itemCount -> 
fromIndex -> toIndex" in korean. 
```

Original issue reported on code.google.com by `sunchi...@gmail.com` on 4 Apr 2013 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
Thanks, this should just be a matter of ordering attributes on those xliff 
strings.
```

Original comment by `caseybur...@google.com` on 4 Apr 2013 at 9:52
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.3.2_b1
```

Original comment by `caseybur...@google.com` on 21 Apr 2013 at 9:41
- Changed state: **Fixed**
